packages/governance/tools/fakeContractGovernor.js

```diff
@@ -0,0 +1,95 @@
+import '../src/types-ambient.js';
+
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+
+// eslint-disable-next-line no-unused-vars
+import { CONTRACT_ELECTORATE } from '../src/contractGovernance/governParam.js';
+
+// @file fakeContractGovernor is a simplified copy of ContractGovernor to
+// simplify testing. It has no electorate, and doesn't try to support
+// legibility. It does try to ensure that the governed contract can get complete
+// support for governance of parameters, apis, and filters. Tests should be able
+// to update parameters directly.
+
+/**
+ * @template {() => {creatorFacet: GovernorFacet<any>, publicFacet: GovernedPublicFacetMethods} } SF Start function of governed contract
+ * @param {ZCF<{
+ *   timer: TimerService,
+ *   governedContractInstallation: Installation<SF>,
+ *   governed: {
+ *     issuerKeywordRecord: IssuerKeywordRecord,
```

Comment:
@turadg,  lint is unhappy with `ZCF`, `TimerService`, and `IssuerKeywordRecord` here, or at least WebStorm highlights them in red. How could I cause lint to run in `tools/`, and what do I import to get these names?

Reply:
f076dbf79db42491526b0b3ed2bc7cadf793d68f shows how to add `tools/`. Then you'll need to a suppression here as in the non-fake contract, https://github.com/Agoric/agoric-sdk/blob/c542cb6eaa52b131c589ad5197373276db6c2452/packages/governance/src/contractGovernor.js#L163-L165

When I tried this I also ran into a problem with `feeMintAccess` from `makeZoeKit`. Looks like that returns a `feeMintAccessRetriever` since #5997.  You might want to rebase this PR on master.

Reply:
The feeMintAccessRetriever is going away in #6726.  Oh, and thanks.  We don't need `feeMintAccess` at all here.

Reply:
Thanks for the magic password.